pkgs/development/libraries/qt-6/modules/qtwebengine.nix

```diff
@@ -131,6 +131,8 @@ qtModule {
   patches = [
     ../patches/qtwebengine-darwin-no-low-latency-flag.patch
     ../patches/qtwebengine-darwin-no-copy-certificate-chain.patch
+    # Don't assume /usr/share/x11, see https://github.com/NixOS/nixpkgs/issues/226484
+    ../patches/qtwebengine-xkb-includes.patch
   ];
 
```

Comment:
Maybe get rid of lines 164-166, since this is now a noop? (I can't put the comment there)

Reply:
Yup, indeed, done!